server.go

```diff
@@ -3078,7 +3078,7 @@ func (s *server) watchChannelStatus() {
 	status := make(map[wire.OutPoint]activeStatus)
 
 	// We'll check in on the channel statuses every 1/4 of the timeout.
-	unchangedTimeout := cfg.InactiveChanTimeout
+	unchangedTimeout := cfg.ChanStatusInterval
```

Comment:
nit: just do the variable name change and this as the first commit?

Reply:
was trying to do it in a way such that the intermediate commits would build :P

Reply:
Haha, yeah I think if you do the `config+server` change it will build 😀